pkg/scheduler/framework/types.go

```diff
@@ -86,6 +86,11 @@ type ClusterEvent struct {
 	ActionType ActionType
 }
 
+// PreEnqueueCheck is a function type. It's used to build functions that
+// run against a Pod and the caller can choose to enqueue or skip the Pod
+// by the checking result.
+type PreEnqueueCheck func(pod *v1.Pod) bool
```

Comment:
I don't think this should be defined here.

Reply:
done.